src/com/company/topic3/Operations.java

```diff
@@ -0,0 +1,50 @@
+package com.company.topic3;
+
+public class Operations {
+    static void printAritmeticOperations(int a, int b) {
+        //int a = 12;
```

Comment:
you can remove the local variables as they are not used anymore in the code. This comment is available for all the cases below.